downstream/assemblies/platform/assembly-planning-installation.adoc

```diff
@@ -10,7 +10,9 @@ ifdef::context[:parent-context: {context}]
 :context: planning
 
 [role="_abstract"]
-To help plan your {PlatformName} installation, review information on the setup installer, system requirements, and supported installation scenarios.
+Red Hat Ansible Automation Platform is supported on both Red Hat Enterprise Linux 8 and Red Hat Openshift. You can use this guide to plan your Red Hat Ansible Automation Platform installation on Red Hat Enterprise Linux 8.
```

Comment:
Don't we run on RHEL9 as well?  Maybe just leave off the version #

Reply:
Good catch. I've removed the version #. Please indicate in the Jira ticket [ https://issues.redhat.com/browse/AAP-3514] whether you approve the change. Thanks!